src/init.cpp

```diff
@@ -1199,12 +1201,6 @@ bool AppInitMain()
     InitSignatureCache();
     InitScriptExecutionCache();
 
-    LogPrintf("Using %u threads for script verification\n", nScriptCheckThreads);
```

Comment:
I was expecting to see this replaced with a `StartScriptCheck();` line; why is that not necessary?

Reply:
I think I forgot to add it. Fixed now.